packages/aws-cdk/lib/init-templates/app/python/README.md

```diff
@@ -0,0 +1,42 @@
+
+Welcome to your CDK Python project!
+
+You should explore the contents of this template. It demonstrates a CDK app with two instances of
+a stack (`HelloStack`) which also uses a user-defined construct (`HelloConstruct`).
+
+The `cdk.json` file tells the CDK Toolkit how to execute your app. It uses a script called `app.sh`
+to do that.
+
+This project is set up like a standard Python project.  The initialization process also creates
+a virtualenv within this project, stored under the .env directory.
+
+After the init process completes, you can use the following steps to get your project set up.
+
+```
+$ source .env/bin/activate
+$ pip install -r requirements.txt
+$ python setup.py develop
+```
+
+At this point you can now synthesize the CloudFormation template for this code.
+
+```
+$ cdk synthesize
+```
+
+You can now begin exploring the source code, contained in the hello directory.
+There is also a very trivial test included that can be run like this:
+
+```
+$ pytest
+```
+
```

Comment:
Workflow for adding a dependency? I am assuming just rerun `pip install -r`, right? Do you need to activate the virtualenv every time?

Reply:
Yes, just add to requirements.txt and return the `pip install -r`.  If you are in a console you need to activate the virtualenv with each console session.  If you are using VS Code, for example, it will just find the virtualenv and use it automatically.

Reply:
Added some text to README.md to describe this.